tests/conftest.py

```diff
@@ -16,16 +15,6 @@ def main_container(dockerc):
     return dockerc.containers(service_names=[MAIN_SERVICE_NAME], stopped=True)[0]
 
 
-@pytest.fixture(scope="session")
-def version_container(dockerc):
-    """Return the version container from the docker composition.
-
-    The version container should just output the version of its underlying contents.
-    """
-    # find the container by name even if it is stopped already
-    return dockerc.containers(service_names=[VERSION_SERVICE_NAME], stopped=True)[0]
-
-
```

Comment:
Why do we remove these tests?

Reply:
@jsf9k Like my above comment, this test wasn't really appropriate for this particular docker container. So the tests were modified in such a way as to keep the original purpose, but be more tailored to this specific container.